devel/apache-arrow/Portfile

```diff
@@ -83,11 +83,6 @@ depends_lib-append \
                     port:zlib \
                     port:zstd
 
-# https://github.com/apache/arrow/pull/35046
-patchfiles-append   patch-io_util.diff
-# https://github.com/apache/arrow/issues/35833
-patchfiles-append   patch-absl-ver.diff
```

Comment:
Can you remove the actual patch files as well?

Reply:
Right, thanks. Done.